nncf/extensions/__init__.py

```diff
@@ -1,5 +1,41 @@
+import enum
 import torch
 
+import os
+
+from nncf.common.utils.registry import Registry
+
+EXTENSIONS = Registry('extensions')
+
+
+class ExtensionsType(enum.Enum):
+    CPU = 0
+    CUDA = 1
+
+
+def _force_build_extensions(ext_type):
+    for class_type in EXTENSIONS.registry_dict.values():
+        if class_type.extension_type() != ext_type:
+            continue
+        class_type.load()
+
+
+def force_build_cpu_extensions():
+    _force_build_extensions(ExtensionsType.CPU)
+
+
+def force_build_cuda_extensions(arch_list=None):
```

Comment:
Should not give an option to the user here to set  - let them set the environment variables on their own.

Reply:
Done